### PR TITLE
uutf-windows.1.0.3

### PR DESCRIPTION
**Pull request by Chimrod**
Update to latest uutf.

Fix https://github.com/ocaml-cross/opam-cross-windows/issues/276

### CODE REVIEW

**Comment by toots** [comment]
Thanks for submiting these and sorry for the delayed response.

I'm not sure why the CI is failing. Have you been able to run it locally?

**Comment by Chimrod** [comment]
No, I have the same error when running the tests locally (but I don’t understand either the cause of the error :( )

File: packages/uutf-windows/uutf-windows.1.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+doc: "https://erratique.ch/software/uutf/doc/"
+dev-repo: "git+https://erratique.ch/repos/uutf.git"
+bug-reports: "https://github.com/dbuenzli/uutf/issues"
+license: ["ISC"]
+tags: ["unicode" "text" "utf-8" "utf-16" "codec" "org:erratique"]
+depends: ["ocaml" {>= "4.03.0"}
+          "ocaml-windows" {>= "4.03.0"}
+          "ocamlfind" {build}
+          "ocamlbuild" {build}
+          "topkg-windows" {build & >= "1.0.3"}]
```
Comment (toots) [inline packages/uutf-windows/uutf-windows.1.0.3/opam:15]:
```suggestion
          "topkg-windows" {build & >= "1.0.3"}]
          "topkg" {build & >= "1.0.3"}]
```

File: packages/uutf-windows/uutf-windows.1.0.3/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
```
Comment (toots) [inline packages/uutf-windows/uutf-windows.1.0.3/opam:15]:
```suggestion
          "topkg-windows" {build & >= "1.0.3"}
```

**Pull request closed without merging**